nbodykit/measurestats.py

```diff
@@ -460,24 +455,34 @@ def project_to_basis(comm, x3d, y3d, edges, los='z', poles=[], symmetric=True):
         xsum.flat += numpy.bincount(multi_index, weights=scratch.flat, minlength=xsum.size)
     
         # take the sum of weights
-        scratch[...] = 1.0
-        # count modes not in singular plane twice
-        if symmetric: scratch[:, nonsingular] = 2.
+        scratch[...] = 1.0 
+        if symmetric: scratch[:, nonsingular] = 2. # count modes not in singular plane twice
```

Comment:
Suppose we are dealing with a pure imaginary x3d, and LOS direction is z.
If we add up positive and negative mu into the same bin, the scratch shall cancel out.